app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -16,6 +16,7 @@ def initialize(targets)
       group_targets_by_ems(targets)
     end
 
+    # Get config information from the config/vmdb.yml file.
```

Comment:
This comment is potentially confusing since the values don't come from vmdb.yml if modified.  Besides, the code clearly pull the options from VMDB::Config, so to me the comment is not needed.